### PR TITLE
refactor components with flexbox grid 

### PR DESCRIPTION
**Pull request by mrbrianhobo**
added flexbox grid for:
- header navbar
- leaderboard page

need to fix random scrolling (i.e. margins)

### CODE REVIEW

**Comment by mrbrianhobo** [comment]
- fixed random scrolling in margins via padding + box-sizing
- added flexbox grid for dashboard page

File: src/components/Card/Card.module.scss [hunk 1]
```diff
@@ -1,11 +1,11 @@
 @import "~styles/variables.scss";
 
 .cardContainer {
-  min-width: 750px;
+/*  min-width: 750px;*/
```
Comment (villanuevawill) [inline src/components/Card/Card.module.scss]: @mrbrianhobo awesome - yeah this should be now based on the column properties.

File: src/components/Header/index.js [hunk 2]
```diff
@@ -27,24 +27,42 @@ const Header = props => {
...
-          </Button>
+        <div className="col-xs-2">
+          <div className="row middle-xs center-xs">Network</div>
         </div>
-      )}
+        {loginStatus ? (
+          <div className="col-xs-offset-5 col-xs-4">
+            <div className="row middle-xs">
+              <Button
+                className="col-xs-offset-2 col-xs-4"
```
Comment (villanuevawill) [inline src/components/Header/index.js]: @mrbrianhobo still here we do not want to put column classes directly on these components (button, etc.). They should be on a wrapper div.

File: src/components/Header/index.js [hunk 3]
```diff
@@ -27,24 +27,42 @@ const Header = props => {
...
+                style="primary"
+                onClick={onCreateClick}
+              >
+                Create New Bounty
+              </Button>
+              <FontAwesomeIcon className="col-xs-2" icon={faBell} />
```
Comment (villanuevawill) [inline src/components/Header/index.js]: same here - you can wrap this in a div.

File: src/components/Header/index.js [hunk 4]
```diff
@@ -27,24 +27,42 @@ const Header = props => {
...
+              <Circle
```
Comment (villanuevawill) [inline src/components/Header/index.js]: same

File: src/containers/DashboardPage/index.js [hunk 5]
```diff
@@ -296,78 +296,90 @@ class DashboardPage extends React.Component {
...
               </div>
-              {/* <div className={`${styles.mySubmissions}`}>{ renderBounties(bountiesData) }</div> */}
-            </div>
-
-            <div className={`${styles.dashboardBodyRight}`}>
-              <div className={`${styles.activity}`}>
-                <Card title="Activity" tabs={notificationTabs}>
-                  {renderNotification(notificationData)}
-                </Card>
+              <div className="col-xs-5">
```
Comment (villanuevawill) [inline src/containers/DashboardPage/index.js]: looking good on this file!

File: src/containers/ExplorerPage/ExplorerPage.module.scss [hunk 6]
```diff
@@ -4,8 +4,6 @@
   width: 100%;
   height: fit-content;
   display: flex;
-  margin-top: 55px;
```
Comment (villanuevawill) [inline src/containers/ExplorerPage/ExplorerPage.module.scss]: nice :)

File: src/containers/ExplorerPage/ExplorerPage.module.scss [hunk 7]
```diff
@@ -4,8 +4,6 @@
   width: 100%;
   height: fit-content;
   display: flex;
```
Comment (villanuevawill) [inline src/containers/ExplorerPage/ExplorerPage.module.scss]: we want to remove this display flex and everything as well right?

File: src/containers/LeaderboardPage/index.js [hunk 8]
```diff
@@ -35,19 +35,21 @@ const LeaderboardPage = props => {
   };
 
   return (
-    <div className={`${styles.leaderboardPage}`}>
-      <div className={`${styles.leaderboardHeader}`}>
-        <div className={`${styles.leaderboardHeaderBody}`}>
-          <Text style="H1" color="white">
-            Leaderboard
-          </Text>
+    <div className={`${styles.leaderboardPage} col-xs`}>
```
Comment (villanuevawill) [inline src/containers/LeaderboardPage/index.js]: I'm confused here. Why does it start off with a col-xs?  Usually a column will have a number on it and will preface a row.

File: src/layout/App/App.module.scss [hunk 9]
```diff
@@ -8,4 +8,6 @@
   width: 100%;
   position: absolute;
   display: flex;
+  padding: 55px 0 0 55px;
```
Comment (villanuevawill) [inline src/layout/App/App.module.scss]: way better :)

**Pull request closed without merging**